libs/optimizer/src/lib/utils/splitSegment.ts

```diff
@@ -0,0 +1,49 @@
+import { LatLonAlt } from '@flyxc/common';
+import { ScoringTrack } from '../optimizer';
+
+export type LatLonAltTime = LatLonAlt & { timeSec: number };
+
+/**
+ * Split a segment into a given number of intervals.
+ * Added points are computed by a linear interpolation
+ * @param from start point of the segment
+ * @param to end point
+ * @param nbIntervals number of intervals in the results. If nbIntervals <= 1
```

Comment:
"If nbIntervals <= 1" ?

Maybe `createSegment` would be a better name?

I find `nbIntervals` confusing, what about `numPoints` instead (>=2)

Reply:
Otherwise, `nbSegments` could be more natural and it does not change the semantic of this argument.